airflow/contrib/hooks/qubole_hook.py

```diff
@@ -175,6 +178,29 @@ def get_jobs_id(self, ti):
             cmd_id = ti.xcom_pull(key="qbol_cmd_id", task_ids=self.task_id)
         Command.get_jobs_id(self.cls, cmd_id)
 
+    def get_redirect_url(self, task, dttm):
+        session = settings.Session()
+        url = ''
+
+        try:
+            conn = BaseHook.get_connection(task.kwargs['qubole_conn_id'])
+            if conn and conn.host:
+                host = re.sub(r'api$', 'v2/analyze?command_id=', conn.host)
+            else:
+                host = 'https://api.qubole.com/v2/analyze?command_id='
+
+            ti = TaskInstance(task=task, execution_date=dttm)
+            qds_command_id = ti.xcom_pull(task_ids=task.task_id, key='qbol_cmd_id')
+
+            url = host + str(qds_command_id) if qds_command_id else ''
+        except Exception as e:
+            print('Could not find the url to redirect. Error: %s' % str(e))
+        finally:
+            session.commit()
```

Comment:
This may throw an AttribueError if session is not properly setup (eg, if settings.Session() returns None)

Reply:
@wrp but we've used a similar piece of code at a lot many places, so its highly unlikely that it'll throw an error only here and not other places.